Makefile

```diff
@@ -33,47 +33,138 @@ FPM_OPTS_RPM := -t rpm \
+el7: toor/el7/usr/lib/systemd/system/marathon.service
+el7: toor/el7/$(PREFIX)/bin/marathon
+el7: marathon.el7.postinst
+	fpm -C toor/el7 --config-files usr/lib/systemd/system/marathon.service \
+		--iteration $(PKG_REL).el7 \
+		--after-install marathon.el7.postinst \
+		$(FPM_OPTS_RPM) $(FPM_OPTS) .
+
+.PHONY: ubuntu-precise
+ubuntu-precise: toor/ubuntu-precise/etc/init/marathon.conf
+ubuntu-precise: toor/ubuntu-precise/etc/init.d/marathon
+ubuntu-precise: toor/ubuntu-precise/$(PREFIX)/bin/marathon
+ubuntu-precise: marathon.postinst
+ubuntu-precise: marathon.postrm
+	fpm -C toor/ubuntu-precise --config-files etc/ --iteration $(PKG_REL).ubuntu1204 \
+		$(FPM_OPTS_DEB) $(FPM_OPTS) .
+
+.PHONY: ubuntu-quantal
+ubuntu-quantal: toor/ubuntu-quantal/etc/init/marathon.conf
+ubuntu-quantal: toor/ubuntu-quantal/etc/init.d/marathon
+ubuntu-quantal: toor/ubuntu-quantal/$(PREFIX)/bin/marathon
+ubuntu-quantal: marathon.postinst
+ubuntu-quantal: marathon.postrm
+	fpm -C toor/ubuntu-quantal --config-files etc/ --iteration $(PKG_REL).ubuntu1210 \
+		$(FPM_OPTS_DEB) $(FPM_OPTS) .
+
+.PHONY: ubuntu-raring
+ubuntu-raring: toor/ubuntu-raring/etc/init/marathon.conf
+ubuntu-raring: toor/ubuntu-raring/etc/init.d/marathon
+ubuntu-raring: toor/ubuntu-raring/$(PREFIX)/bin/marathon
+ubuntu-raring: marathon.postinst
+ubuntu-raring: marathon.postrm
+	fpm -C toor/ubuntu-raring --config-files etc/ --iteration $(PKG_REL).ubuntu1304 \
+		$(FPM_OPTS_DEB) $(FPM_OPTS) .
+
+.PHONY: ubuntu-saucy
+ubuntu-saucy: toor/ubuntu-saucy/etc/init/marathon.conf
+ubuntu-saucy: toor/ubuntu-saucy/etc/init.d/marathon
+ubuntu-saucy: toor/ubuntu-saucy/$(PREFIX)/bin/marathon
+ubuntu-saucy: marathon.postinst
+ubuntu-saucy: marathon.postrm
+	fpm -C toor/ubuntu-saucy --config-files etc/ --iteration $(PKG_REL).ubuntu1310 \
+		$(FPM_OPTS_DEB) $(FPM_OPTS) .
+
+.PHONY: ubuntu-trusty
+ubuntu-trusty: toor/ubuntu-trusty/etc/init/marathon.conf
+ubuntu-trusty: toor/ubuntu-trusty/etc/init.d/marathon
+ubuntu-trusty: toor/ubuntu-trusty/$(PREFIX)/bin/marathon
+ubuntu-trusty: marathon.postinst
+ubuntu-trusty: marathon.postrm
+	fpm -C toor/ubuntu-trusty --config-files etc/ --iteration $(PKG_REL).ubuntu1404 \
+		$(FPM_OPTS_DEB) $(FPM_OPTS) .
+
+.PHONY: ubuntu-utopic
+ubuntu-utopic: toor/ubuntu-utopic/etc/init/marathon.conf
+ubuntu-utopic: toor/ubuntu-utopic/etc/init.d/marathon
+ubuntu-utopic: toor/ubuntu-utopic/$(PREFIX)/bin/marathon
+ubuntu-utopic: marathon.postinst
+ubuntu-utopic: marathon.postrm
+	fpm -C toor/ubuntu-utopic --config-files etc/ --iteration $(PKG_REL).ubuntu1410 \
+		$(FPM_OPTS_DEB) $(FPM_OPTS) .
+
+.PHONY: debian-wheezy-75
+debian-wheezy-75: toor/debian-wheezy-75/etc/init/marathon.conf
+debian-wheezy-75: toor/debian-wheezy-75/etc/init.d/marathon
+debian-wheezy-75: toor/debian-wheezy-75/$(PREFIX)/bin/marathon
+debian-wheezy-75: marathon.postinst
+debian-wheezy-75: marathon.postrm
+	fpm -C toor/debian-wheezy-75 --config-files etc/ --iteration $(PKG_REL).debian75 \
+		$(FPM_OPTS_DEB) $(FPM_OPTS) .
+
```

Comment:
What is the reason for having multiple debian wheezy packages? I think we should be fine with just a single package for both 7.5 and 7.7.